### ISSUE TITLE
Support for FAB(Floating Action Button)

### ISSUE THREAD

**Issue by benitogonzalezh** [fluttercommunity/flutter_webview_plugin#812]
It would be very useful implement a Floating Action Button in the args of the plugin.

**Comment by bagus-repository** [comment]
yeah I want to know if a Floating Action Button can be implemented into WebviewScaffold()

**Comment by sboicu** [comment]
I also try to have such feature but I was not able to find one.
I need it mainly for a refresh button whit I do not want to have it under a app bar or bottom navigation.
